examples/build_sample_experiment.py

```diff
@@ -20,15 +20,15 @@
 }
 
 
-def tile_opener(toc_path, tile, ext):
+def tile_opener(toc_path, tile, file_ext):
     tile_basename = os.path.splitext(toc_path)[0]
     return open(
         "{}-Z{}-H{}-C{}.{}".format(
```

Comment:
Is this meant to be starfish specific, or just for this experiment? I ask because we haven't specified that tiles need to be saved with specific naming conventions in the spec.